tests/utils.rs

```diff
@@ -19,7 +19,29 @@ pub fn clone_out_test(source: &str) -> (tempdir::TempDir, String) {
     (tmpdir, path)
 }
 
-/// Execute localc cargo command, includes `--manifest-path`, expect command failed
+/// Add directory
+pub fn setup_alt_registry_config(path: &std::path::Path) {
+    fs::create_dir(path.join(".cargo")).expect("failed to create .cargo directory");
+    fs::copy(
+        "tests/fixtures/alt-registry-cargo-config",
+        path.join(".cargo").join("config"),
+    )
+    .unwrap_or_else(|err| panic!("could not copy test cargo config: {}", err));
+    // Workaround for https://github.com/killercup/cargo-edit/issues/339
```

Comment:
Thanks! It suffices to init the repo instead of cloning it, so I've done that to avoid issues like https://github.com/killercup/cargo-edit/issues/333 when cloning as well as fetching.